config/initializers/10_aws_sdk.rb

```diff
@@ -0,0 +1,6 @@
+if (not AWSConfig.config.aws_access_id.blank?) && (not AWSConfig.config.aws_secret_key.blank?)
```

Comment:
__[rubocop]__
* Style/Not: Use `!` instead of `not`. [:ref](https://github.com/bbatsov/ruby-style-guide#bang-not-not)
* Style/Not: Use `!` instead of `not`. [:ref](https://github.com/bbatsov/ruby-style-guide#bang-not-not)